src/editor/new_exercise.ml

```diff
@@ -0,0 +1,230 @@
+(* This file is part of Learn-OCaml.
+ *
+ * Copyright (C) 2019 OCaml Software Foundation.
+ * Copyright (C) 2016-2018 OCamlPro.
+ *
+ * The main authors of the editor part is the pfitaxel team see 
+ * https://github.com/pfitaxel/learn-ocaml-editor for more information
+ * 
+ * Learn-OCaml is distributed under the terms of the MIT license. See the
+ * included LICENSE file for details. *)
+
+open Js_of_ocaml
+open Dom_html
+open Js_utils
+open Learnocaml_common
+open Editor_lib
+open Learnocaml_config
+open Learnocaml_data
+open Learnocaml_data.Editor
+open Learnocaml_data.Exercise.Meta
+
+module StringMap = Map.Make (String)
+(*
+(* Internationalization *)
+let () = Translate.set_lang ()
```

Comment:
Are you planning to uncomment this code?

Reply:
Hi @yurug, I guess so… (but this PR is not yet ready for review !-)